sample/host/src/AuthenticatedLayout.tsx

```diff
@@ -22,20 +25,35 @@ export default function AuthenticatedLayout() {
 
     const navigationItems = useNavigationItems();
 
```

Comment:
renderLink. renderMenu, renderItem and renderSection can be put outside of the component. This will increase performance as the methods won't have to be created for each instance of the component and we won'thave to useCallback any of those functions